### PR TITLE
Replace slack channel link with discord server link

### PR DESCRIPTION
**Pull request by sahanHe**
## Purpose
https://github.com/ballerina-platform/ballerina-standard-library/issues/3463
## Checklist
- [x] Linked to an issue
- [ ] Updated the changelog
- [ ] Added tests
- [ ] Updated the spec

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -2,7 +2,7 @@ org.gradle.caching=true
 group=io.ballerina.stdlib
 version=2.2.3-SNAPSHOT
```
Comment (niveathika) [inline gradle.properties:3]: Version should change along with distribution update in toml file

Comment (sahanHe) [reply gradle.properties]: updated in [5d3278c](https://github.com/ballerina-platform/module-ballerina-xslt/pull/424/commits/5d3278cdab8a1c88a6a0814a90ea571aebeda4e9)

File: ballerina/Ballerina.toml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 [package]
 org = "ballerina"
 name = "xslt"
-version = "2.2.3"
+version = "2.3.0"
```
Comment (niveathika) [inline ballerina/Ballerina.toml:4]: distribution still points to 2201.0.4

Comment (niveathika) [reply ballerina/Ballerina.toml:4]: This file gets updated from a template. There is no point updating here, since the next build will ovveride

Comment (sahanHe) [reply ballerina/Ballerina.toml:4]: Yes, the template was updated in [78846df](https://github.com/ballerina-platform/module-ballerina-xslt/pull/424/commits/78846dfac0f2451b5a5e41e7eb1244a5ebfffa67)

**Review by niveathika — APPROVED** [review]

**Pull request merged**